x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_detail_panel/data_stream_detail_panel.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiFlyout,
+  EuiFlyoutHeader,
+  EuiTitle,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { SectionLoading, SectionError, Error } from '../../../../components';
+import { useLoadDataStream } from '../../../../services/api';
+
+interface Props {
+  dataStreamName: string;
+  onClose: () => void;
+}
+
+export const DataStreamDetailPanel: React.FunctionComponent<Props> = ({
```

Comment:
Do you think it's worth adding a comment to this file that it's not currently being used and is a WIP?